pkg/services/ngalert/api/api_prometheus.go

```diff
@@ -149,17 +147,10 @@ func (srv PrometheusSrv) RouteGetRuleStatuses(c *models.ReqContext) response.Res
 		if !ok {
 			continue
 		}
-		var queryStr string
-		encodedQuery, err := json.Marshal(rule.Data)
-		if err != nil {
-			queryStr = err.Error()
-		} else {
-			queryStr = string(encodedQuery)
-		}
 		alertingRule := apimodels.AlertingRule{
 			State:       "inactive",
 			Name:        rule.Title,
-			Query:       queryStr,
+			Query:       ruleToQuery(srv.log, rule),
```

Comment:
This is the _actual_ change of the PR - the rest of the lines are just tests and fixtures.